executor/show_test.go

```diff
@@ -1375,4 +1375,13 @@ func (s *testSuite5) TestShowTemporaryTable(c *C) {
 		"  KEY `b` (`b`)\n" +
 		") ENGINE=memory DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin ON COMMIT DELETE ROWS"
 	tk.MustQuery("show create table t5").Check(testkit.Rows("t5 " + expect))
+
+	tk.MustExec("set tidb_enable_noop_functions=true")
+	tk.MustExec("create temporary table t6 (i int primary key, j int)")
```

Comment:
Try to create another table with auto-id.

Reply:
done

Reply:
Does it need to check the result of `show create table`?

Reply:
> Does it need to check the result of `show create table`?

done